### PR TITLE
Add OORT Devnet (Testnet)

### PR DESCRIPTION
**Pull request by molecula451**
### Description

Add OORT DevNet (Testnet)

### CODE REVIEW

File: chains/oortdevnet/logo.svg
Comment (jmrossy) [inline chains/oortdevnet/logo.svg:1]: Can you please remove the extra whitespace around this logo and also minify it? These online tools can help:
https://jsfiddle.net/knLbjc9a/4/
https://jakearchibald.github.io/svgomg/

File: chains/oortdevnet/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  family: other
+  name: OORT Dev Explorer
+  url: https://dev-scan.oortech.com
+blocks:
+  confirmations: 1
+  estimateBlockTime: 1
+  reorgPeriod: 1
+chainId: 9700
+deployer:
+  email: novacypher@oortech.com
```
Comment (jmrossy) [inline chains/oortdevnet/metadata.yaml:12]: Thanks for including this. Would you mind also setting the `name` field here?

File: chains/oortdevnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  family: other
+  name: OORT Dev Explorer
+  url: https://dev-scan.oortech.com
```
Comment (jmrossy) [inline chains/oortdevnet/metadata.yaml:5]: an `apiUrl` will also be required here

**Review by jmrossy — COMMENTED** [review]
Thanks for the contribution! Just a few things to fix

**Comment by jmrossy** [comment]
@molecula451 I see this PR is still in Draft mode. Was that intentional or would you like me to take another look?

**Comment by molecula451** [comment]
hey @jmrossy this was intentinal, re-opening this ASAP, im checking somethings on the metadata, re-tagging you

**Comment by jmrossy** [comment]
@molecula451 no rush, take your time :)
I'll keep an eye out for updates here

**Comment by molecula451** [comment]
OORT is already on mainnet, i think this PR could be re-do for testnet

**Pull request closed without merging**